plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProvider.java

```diff
@@ -514,6 +518,10 @@ private Instance createInstance(final String containerName,
                 volumes = commonMachineSystemVolumes;
                 env = new ArrayList<>(commonMachineEnvVariables);
             }
+
+            final long machineMemory = machine.getConfig().getLimits().getRam() * 1024 * 1024;
+            final long machineMemorySwap = (long)(memorySwapMultiplier == -1 ? -1 : machineMemory * memorySwapMultiplier);
```

Comment:
As for me it is better to minimize the cast scope `(long)(machineMemory * memorySwapMultiplier)`